ecommerce/frontend/src/components/TrendingGrid.js

```diff
@@ -1,23 +1,24 @@
 import './TrendingGrid.less'
+import { Link } from 'react-router-dom'
+
+export const TrendingGrid = trendingProducts => {
+    var gallerytItemNumber = 0
+    function gallerytItemNo() {
+        gallerytItemNumber += 1
+        return gallerytItemNumber
+    }
```

Comment:
No, please do not use any "hack" like this. The proper way of doing this is by doing 

```js
products.map((product, index) => {
...
})
```

and using `index` for the classNames